extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -493,6 +494,8 @@ export const RoundTripSqlDbNotPresentCondition = '\'$(NetCoreBuild)\' != \'true\
 export const DacpacRootPath = '$(DacPacRootPath)';
 export const ProjJsonToClean = '$(BaseIntermediateOutputPath)\\project.assets.json';
 
+export const defaultOutputPath = '.\\bin\\Debug\\';
```

Comment:
should this be a non-const that's constructed with `path.join()` to make sure the slashes work on all plats?

Reply:
Updated to use `path.join()`. It was getting converted with `utils.getPlatformSafeFileEntryPath()` in project.ts, so it was working on mac, but good to update it in case it's used somewhere else in the future so it doesn't get missed.